util/test_db.go

```diff
@@ -0,0 +1,100 @@
+package util
```

Comment:
A best practice is to indicate shared code for testing only by package names such as `footest`.  So this could be called `utiltest`.

https://stackoverflow.com/a/56412373/2308716

Reply:
As this PR will be replaced by more uptodated code so I applied your suggested changes in our repository and they will be avail in a future PR.